webserver/web-interface/src/components/chart/ShotChart.tsx

```diff
@@ -152,10 +149,26 @@ function addDataPointToChartData(chartData, dataPoint, maxLength, chartRef) {
   /* eslint-enable no-param-reassign */
 }
 
-function Chart({ data, newDataPoint, maxLength }) {
-  const chartRef = useRef(null);
+export interface ShotChartProps {
+  data?: ShotSnapshot[],
+  newDataPoint?: ShotSnapshot,
+  maxLength?: number,
+  onHover?: (datapoint: ShotSnapshot) => void,
+}
+
+function ShotChart({
+  data, newDataPoint, maxLength, onHover,
+}: ShotChartProps) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const chartRef = useRef<any>();
```

Comment:
nit: this can be `ChartJSOrUndefined<"line", any, unknown>` I think (though I guess it still has any so meh, doesn't matter too much)